Source/WebKit/UIProcess/ios/WKTextInteractionWrapper.mm

```diff
@@ -209,8 +209,10 @@ - (void)prepareToMoveSelectionContainer:(UIView *)newContainer
         for (UIView *subview in newContainer.subviews)
             [viewsBeforeInstallingInteraction addObject:subview];
 
-        // Calling these delegate methods tells the display interaction to remove and reparent all internally
-        // managed views (e.g. selection highlight views, selection handles) in the new selection container.
+        // When the display interaction is in the activated state, calling these delegate methods tells it
+        // to remove and reparent all internally managed views (e.g. selection highlight views, selection
+        // handles) in the new selection container.
+        [displayInteraction setActivated:YES];
```

Comment:
Maybe this could use `-activateSelection` instead? (So that it will fall back to `[_textInteractionAssistant activateSelection];` in the case where BEK is not used).